poollet/machinepoollet/controllers/controllers_suite_test.go

```diff
@@ -54,7 +54,7 @@ var (
 const (
 	eventuallyTimeout    = 3 * time.Second
 	pollingInterval      = 50 * time.Millisecond
-	consistentlyDuration = 1 * time.Second
+	consistentlyDuration = 10 * time.Second
```

Comment:
10 seconds is quite a long time IMO. As the polling interval is currently set to 50 milliseconds maybe we should stick with 3 seconds (same as the eventually timeout).

Reply:
done